app/src/main/java/studyBuddy/timemanagement/StrategyRecord.java

```diff
@@ -0,0 +1,42 @@
+package studyBuddy.timemanagement;
+
+import androidx.annotation.NonNull;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+import java.util.Date;
+
+/**
+ * Stores all information necessary to represent a session at some point in time
+ */
+public class StrategyRecord {
+    public final SessionType type;
+    public final String name;
+    public final Date start;
+    public final Date end;
+    public final double percentProductive;
+
+    public StrategyRecord(Strategy strategy, String name,
```

Comment:
Should this take a session object?

Reply:
Oh yea that's a much better idea, that way we could get all of the other info on a completed session. i can get that implemented once the session branch is merged